packages/amplify-graphql-schema-generator/API.md

```diff
@@ -90,6 +90,12 @@ export abstract class DataSourceAdapter {
     vpcSchemaInspectorLambda: string | undefined;
 }
 
+// @public (undocumented)
+export type DatasourceConfig = {
```

Comment:
nit Not a huge deal, but it'd be nice to keep it consistent with the naming conventions of the CDK construct. We've been capitalizing it there as `DataSource` rather than `Datasource`